execution/scan/matrix_selector.go

```diff
@@ -54,17 +55,19 @@ type matrixSelector struct {
 	extLookbackDelta int64
 }
 
-// NewMatrixSelector creates operator which selects vector of series over time.
+// NewRangeVectorFunction creates operator which selects vector of series over time.
```

Comment:
Looks like the comment does not match the function anymore :)

Reply:
thanks! fixed it!